star-rx72n-firmware/lib/rx_fec/inc/rx_fec.h

```diff
@@ -0,0 +1,265 @@
+/**
+ * @file rx_fec.h
+ * @brief Forward Error Correction (FEC) Codec for RX72N
+ *
+ * Implements NASA-standard K=7 convolutional encoder and Viterbi decoder.
+ * This is a C port of the Go FEC implementation in star-gateway for
```

Comment:
Why are we using Viterbi, is `BCJR` not better? I know that BCJR requires more computation power, but we have a good chip, and we aren't utilizing much of its processing power in this application. The tradeoff might be worth it for us–I know that viterbi is often "good enough," but I think we should use BCJR if we can.